docs/make.jl

```diff
@@ -26,12 +32,23 @@ makedocs(sitename="FMI.jl",
                 "Multithreading" => "examples/multithreading.md"
                 "Multiprocessing" => "examples/multiprocessing.md"
             ]
-            "Library Functions" => "library.md"
-            "FMI Tool Information" => "fmi-tool-info.md"
+            "User Level API - FMI.jl" => "library.md"
+            "Developer Level API" => Any[
+                "fmi version independent content" => Any[
+                    "fmi_lowlevel_library_types.md",
+                    "fmi_lowlevel_library_constants.md",
+                    "fmi_lowlevel_library_functions.md"
+                ],
+                "FMI2 specific content" => "fmi2_lowlevel_librarys.md",
+                "FMI3 specific content" => "fmi3_lowlevel_librarys.md"
+            ]
+            "API Index" => "index_library.md"
+            "FMI Tool Information" => "fmi-tool-info.md"            
             "Related Publication" => "related.md"
             "Contents" => "contents.md"
-            ]
-         )
+            hide("Deprecated" => "deprecated.md")
```

Comment:
all exported funtions need to be documented. The coresponding file will be hidden (might be removable by removing docstrings from deprececated functions)